src/main/java/com/twilio/oai/resolver/csharp/CsharpCodegenParameterDataTypeResolver.java

```diff
@@ -0,0 +1,62 @@
+package com.twilio.oai.resolver.csharp;
+
+import com.twilio.oai.StringHelper;
+import com.twilio.oai.common.ApplicationConstants;
+import com.twilio.oai.resolver.IConventionMapper;
+import com.twilio.oai.resolver.common.CodegenParameterDataTypeResolver;
+import org.openapitools.codegen.CodegenParameter;
+
+import java.util.Optional;
+
+public class CsharpCodegenParameterDataTypeResolver extends CodegenParameterDataTypeResolver {
+
+    private final IConventionMapper mapper;
+
+    private final CsharpSerializer csharpSerializer;
+    public CsharpCodegenParameterDataTypeResolver(IConventionMapper mapper, CsharpSerializer csharpSerializer) {
+        super(mapper);
+        this.mapper = mapper;
```

Comment:
Same thing about `mapper` being defined in the parent class.

Reply:
resolved